packages/material-ui-lab/src/Pagination/usePagination.test.js

```diff
@@ -0,0 +1,128 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { assert } from 'chai';
+import { usePagination } from '.';
+
+describe('usePagination', () => {
+  it('has one page by default', () => {
+    const { items } = renderHook(() => usePagination()).result.current;
+    assert.strictEqual(items.length, 3);
+    assert.strictEqual(items[1].page, 1);
+  });
+
+  it('has disabled previous & next buttons by default', () => {
+    const { items } = renderHook(() => usePagination()).result.current;
+    assert.strictEqual(items[0].type, 'previous');
```

Comment:
```suggestion
    expect(items[]).to.have.property('type', 'previous')
```

two advantages: 
- better fail message if `type` doesn't match
- better fail message if `items[0]` is nullish